tests/tls_e2e/README.md

```diff
@@ -0,0 +1,34 @@
+Unittests Transport Layer Security end-2-end scenarios
+=====================
+
+Test scenarios:
+
+In this test, we have three test hooks for enabling test scenarios.
```

Comment:
How about rephrasing to: 
Three test hooks are available for enabling test scenarios.

Reply:
Changed as suggested